Breeze-CSS/breeze.css

```diff
@@ -0,0 +1,585 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700;800&display=swap");
+
+* {
+  font-family: "Open Sans", sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --color-1: #1f1f27;
+  --color-2: #f3f4f6;
+}
+
+
+/* Avatar */
+
+
+
+.avatar-sm {
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
```

Comment:
code repetition: border-radius:50% repeated 5 times in your code use CSS variable to avoid it